src/cli/mod.rs

```diff
@@ -529,3 +532,7 @@ fn read_to_base64(path: PathBuf) -> anyhow::Result<String> {
     use base64::{engine::general_purpose, Engine};
     Ok(general_purpose::STANDARD.encode(buffer))
 }
+
+fn parse_json(src: &str) -> Result<serde_json::Value, String> {
```

Comment:
in the cli all returns should be `anyhow::Result<T>`.
Then you should be able to get rid of the `.map_err`.